### PR TITLE
[Filter/Example] update multi objects detection @open sesame 10/23 19:05

### PR DESCRIPTION
**Pull request by helloahn**
Multi-Output Tensor Example: detect the multiple objects through ssd model.

detected

**Self evaluation:**
1. Build test: [*]Passed [ ]Failed [ ]Skipped
2. Run test: [*]Passed [ ]Failed [ ]Skipped

Signed-off-by: HyoungjooAhn <hello.ahnn@gmail.com>

### CODE REVIEW

File: nnstreamer_example/example_object_detection/nnstreamer_example_object_detection.cc [hunk 1]
```diff
@@ -0,0 +1,738 @@
...
+typedef struct
+{
+  GMainLoop *loop; /**< main event loop */
+  GstElement *pipeline; /**< gst pipeline for data stream */
+  GstBus *bus; /**< gst bus for data pipeline */
+  gboolean running; /**< true when app is running */
+  GMutex mutex; /**< mutex for processing */
+  TFLiteModelInfo tflite_info; /**< tflite model info */
+  CairoOverlayState overlay_state;
+    std::vector < DetectedObject > detected_objects;
```
Comment (again4you) [inline nnstreamer_example/example_object_detection/nnstreamer_example_object_detection.cc]: Fix the indentation.

File: nnstreamer_example/example_object_detection/nnstreamer_example_object_detection.cc [hunk 2]
```diff
@@ -0,0 +1,738 @@
...
+      break;
+    }
+  }
+}
+
+/**
+ * @brief Main function.
+ */
+int
+main (int argc, char **argv)
```
Comment (again4you) [inline nnstreamer_example/example_object_detection/nnstreamer_example_object_detection.cc]: Which is the `standard` expression about pointer?
- `char **argv` or `char ** argv`

Comment (helloahn) [reply nnstreamer_example/example_object_detection/nnstreamer_example_object_detection.cc]: I raised #686 for discussion about this problem.

File: nnstreamer_example/example_object_detection/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# demo app
+pkg_check_modules(cairo_pkg REQUIRED cairo)
```
Comment (ohsewon) [inline nnstreamer_example/example_object_detection/CMakeLists.txt]: Don't we have to add cairo package into 'debian/control' build-depends?

File: nnstreamer_example/example_object_detection/nnstreamer_example_object_detection.cc [hunk 4]
```diff
@@ -0,0 +1,738 @@
...
+ * @author	HyoungJoo Ahn <hello.ahn@samsung.com>
+ * @bug		No known bugs.
+ *
+ * Run example :
+ * Before running this example, GST_PLUGIN_PATH should be updated for nnstreamer plug-in.
+ * $ export GST_PLUGIN_PATH=$GST_PLUGIN_PATH:<nnstreamer plugin path>
+ * $ ./nnstreamer_example_object_detection
+* 
+ * Required model and resources are stored at below link
+ * https://github.com/nnsuite/testcases/tree/master/DeepLearningModels/tensorflow-lite/ssd_mobilenet_v2_coco
```
[the remark this answers is not in the document]
Comment (helloahn) [reply nnstreamer_example/example_object_detection/nnstreamer_example_object_detection.cc]: I updated the script for download at `cmake`. please check it

File: nnstreamer_example/example_object_detection/CMakeLists.txt (hunk 3 above)
Comment (helloahn) [reply nnstreamer_example/example_object_detection/CMakeLists.txt]: https://github.com/nnsuite/nnstreamer/blob/f62e5c5f71c445049d37114f3625a13576bb4f18/debian/control#L9

it's updated. thanks :)

**Pull request merged**